app/docs/0.13.x/auth.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Authentication Reference
+---
+
+# Authentication Reference
+
+Client access to upstream API services is typically controlled by the application and configuration of 
+Kong [authentication plugins][plugins]. 
+
+## Generic authentication
+
+The most common scenario is to require authentication and to not allow access for any unauthenticated request.
+To achieve this any of the authentication plugins can be used. The generic scheme/flow of those plugins
+works as follows:
+
+1. Apply an auth plugin to an api, or globally (you cannot apply one on consumers)
+2. Create a `consumer` entity
+3. Provide the consumer with authentication credentials for the specific authentication method
+4. Now whenever a request comes in Kong will check the provided credentials (depends on the auth type) and
+it will either block the request if it cannot validate, or add consumer and credential details
+in the headers and forward the request
+
+The generic flow above does not always apply, for example when using external authentication like LDAP,
+then there is no consumer to be identified, and only the credentials will be added in the forwarded headers.
+
+The authentication method specific elements and examples can be found in each [plugin's documentation][plugins].
+
+## Consumers
+
+The easiest way to think about consumers is to map them one-on-one to users. Yet, to Kong this does not matter.
+The core principle for consumers is that you can attach plugins to them, and hence customize request behaviour.
+So you might have mobile apps, and define one consumer for each app, or version of it. Or have a consumer per
+platform, e.g. an android consumer, an iOS consumer, etc.
+
+It is an opaque concept to Kong and hence they are called "consumers" and not "users".
+
+## Anonymous Access
+
+Prior to Kong 0.10.x, a given API could be configured to allow **only** authenticated access (by applying an
+[auth plugin][plugins]) or **only** anonymous access - it was not possible to have a given API allow some 
+users to be authenticated and others to access anonymously.
+
+Kong 0.10.x adds the ability to configure a given API to allow **both** authenticated **and** anonymous access. 
```

Comment:
Same here - "Kong 0.10.x adds..." - would be better to remove that and just state "Kong has the ability to configure a given API..."